Islandora/islandora_solution_pack_audio
# 7.x islandora 1499

**JIRA Ticket**: https://jira.duraspace.org/browse/ISLANDORA-1499

Release PR: #132
# What does this Pull Request do?

Allows the site admin to configure the quality of audio MP3 derivatives. Currently this is set to 5, which is worse quality than even LAME suggests as a default. 
# What's new?

Added a configuration variable to the admin form to set the LAME variable bitrate quality value. Validate that the input is numeric and in the valid range. Default to 5 if no value is entered (preserving existing behaviour). 
# How should this be tested?
- Make an audio derivative. Notice the file size (or install MediaInfo and see that the encoding variables include -V5)
- After installing this update, without changing the config, ensure that existing behaviour is preserved.
- Check that you can configure the quality value between 0 and 9.999 and the resulting file is of appropriate size. 
- Try to inject something nasty (since what you enter will be run as a shell command) and see that it won't let you. 
# Additional Notes:

Any additional information that you think would be helpful when reviewing this PR.

Example:
- Does this change require documentation to be updated?  Yes. README is updated; tagging the documenter. 
- Does this change add any new dependencies? No. 
- Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)? No. 
- Could this change impact execution of existing code? No. 
# Interested parties

@Islandora/7-x-1-x-committers  please take a gander.
